sharkfuser/include/fusili/logging.h

```diff
@@ -52,6 +54,141 @@ struct [[nodiscard]] ErrorObject {
   bool operator!=(ErrorCode compareCode) const { return code != compareCode; }
 };
 
+// Utility function that returns true if an ErrorObject represents a successful
+// result.
+inline bool isOk(ErrorObject Result) { return Result.isOk(); }
+
+// Utility function that returns true if an ErrorObject represents an
+// unsuccessful result.
+inline bool isError(ErrorObject Result) { return Result.isFailure(); }
```

Comment:
Having it called `isFailure` one place and `isError` another is confusing. Can we call it either `isError` or `isFailure` throughout to be consistent?  (I'm OK if you want to change the ErrorObject method to `isError` to match this, or vice-versa.